config/database.yml.example

```diff
@@ -1,18 +0,0 @@
-default: &default
```

Comment:
I couldn't run server locally so I renamed this file. But now I see that I had to create a new one. Actually can you explain the purpose of that file?